scripts/hud/cgaz.ts

```diff
@@ -691,18 +691,20 @@ class CgazHandler {
 
 		this.clearZones(this.primeZones);
 
-		if (this.primeEnable) {
+		if (this.primeEnable && phyMode !== DefragAPI.DefragPhysics.VTG) {
 			const snapOffset = MomMath.remapAngle(
 				(bSnapShift ? 0 : Math.PI * 0.25 * (rightMove > 0 ? -1 : 1)) - viewAngle
 			);
 
 			if (speed > this.primeMinSpeed) {
+				const bUseSnapAccel = !(
```

Comment:
Don't use hungarian notation. This should just be `useSnapAccel`.

Reply:
old habit from unreal